tests/test_wsireader.py

```diff
@@ -37,35 +37,38 @@
 _, has_codec = optional_import("imagecodecs")
 has_tiff = has_tiff and has_codec
 
-FILE_KEY = "wsi_img"
-FILE_URL = testing_data_config("images", FILE_KEY, "url")
-base_name, extension = os.path.basename(f"{FILE_URL}"), ".tiff"
-FILE_PATH = os.path.join(os.path.dirname(__file__), "testing_data", "temp_" + base_name + extension)
+TIFF_KEY = "wsi_generic_tiff"
+TIFF_URL = testing_data_config("images", TIFF_KEY, "url")
+TIFF_PATH = os.path.join(os.path.dirname(__file__), "testing_data", f"temp_{TIFF_KEY}.tiff")
```

Comment:
could you please come up with a more specific name, otherwise the variable name will be changed back and forth again in the future

Reply:
Sure, what do you think of using the full name as the prefix: `WSI_GERERIC_TIFF`
```python
WSI_GERERIC_TIFF_KEY = "wsi_generic_tiff"
WSI_GERERIC_TIFF_URL = testing_data_config("images", TIFF_KEY, "url")
WSI_GERERIC_TIFF_PATH = os.path.join(os.path.dirname(__file__), "testing_data", f"temp_{TIFF_KEY}.tiff")
```